gapis/service/path/path.proto

```diff
@@ -152,6 +152,7 @@ message Events {
     bool push_user_markers = 8;
     bool pop_user_markers = 9;
     bool framebuffer_observations = 10;
+    bool all_commands = 11;
```

Comment:
I don't understand the purpose of this event. You already have `service.Capture.num_commands` - what more does this tell you?